scripts/!mods_preload/msu_mod_weapon.nut

```diff
@@ -164,6 +164,11 @@ gt.MSU.modWeapon <- function ()
 			return _only ? this.m.WeaponType == _t : (this.m.WeaponType & _t) != 0;
 		}
 
+		o.isAllWeaponTypes <- function( _t )
+		{
+			return _t & this.m.WeaponType == _t;
```

Comment:
Just convert this to `this.m.WeaponType & _t == t;` to be the same as other functions, or it'll bug me :P